pkgs/development/tools/heroku/default.nix

```diff
@@ -17,10 +17,13 @@ stdenv.mkDerivation rec {
     sha256 = "1d472vm37lx5nyyaymjglavisb1mkyzbjglzjp5im7wjfifvsd29";
   };
 
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```

Comment:
If I'm reading the docs correctly, `buildPhase` should do nothing unless a makefile exists. You may set `dontBuild` or `buildPhase = ":"` to specifically disable it. A little cleaner, IMO, but no big deal.